src/utils.js

```diff
@@ -51,7 +51,7 @@ const argumentError = ({ expected = '', received, extraInfo = '' }) => {
   try {
     return new TypeError(`${'You should provide ' + expected}${'\n' + extraInfo}${'\nReceived: ' + JSON.stringify(received)}`)
   } catch (err) {
-    if (err.message === 'Converting circular structure to JSON') {
+    if (err.message.indexOf('Converting circular structure to JSON') > -1) {
```

Comment:
I updated this line because apparently on newer browsers the error message itself includes more details on the property having a circular dependency.

![image](https://user-images.githubusercontent.com/16447522/131233912-26b2b9d0-a7f5-4667-9933-ece4d61646a8.png)

Reply:
I noticed that also. Thank you for the update!